apps/web/instrumentation-client.ts

```diff
@@ -43,6 +44,29 @@ if (process.env.NODE_ENV === "production") {
   });
 }
 
+if (process.env.VERCEL_BOTID_ENABLED === "true") {
```

Comment:
I think it should be prefixed by __NEXT_PUBLIC__ so that it is available at client side.